### PR TITLE
ci: skip tests if PR only touches docs

### PR DESCRIPTION
**Pull request by mcansh**
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-based-on-files-changed-in-a-pull-request-1

closes REM-1072

### CODE REVIEW

**Review by MichaelDeBoey — APPROVED** [review]

**Comment by chaance** [comment]
OH THANK GOD I was just thinking about asking you about this today 😅 🥳

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -11,7 +11,10 @@ on:
     paths-ignore:
       - "docs/**"
       - "**/README.md"
-  pull_request: {}
+  pull_request: 
+    paths-ignore:
+      - "docs/**"
+      - "**/README.md"
```
Comment (chaance) [inline .github/workflows/test.yml:17]: Should we go ahead an ignore all markdown files? 
```suggestion
      - "**/*.md"
```

**Review by chaance — APPROVED** [review]
SHIP IT

**Review by kentcdodds — APPROVED** [review]
Perfecto 👨‍🍳💋

**Pull request merged**